hedera-node/hedera-mono-service/src/main/java/com/hedera/services/context/properties/BootstrapProperties.java

```diff
@@ -56,12 +57,37 @@ public final class BootstrapProperties implements PropertySource {
                 }
                 return in;
             };
-    private static ThrowingStreamProvider fileStreamProvider =
+    static final ThrowingStreamProvider DEFAULT_FILE_STREAM_PROVIDER =
             loc -> Files.newInputStream(Paths.get(loc));
 
+    private final EnvironmentProvider envVarsProvider;
+
     @Inject
     public BootstrapProperties() {
-        /* No-op */
+        this(
```

Comment:
I think these changes should be separated from this PR. Once this PR is merged a new PR can be opened for these changes.